internal/context/init.go

```diff
@@ -32,18 +32,20 @@ func Init(projectName string, outDir string) *config.ZeroProjectConfig {
 
 	projectConfig := defaultProjConfig(projectName)
 	promptProjectName(projectName, &projectConfig)
+	promptPushRepoUpstream(&projectConfig)
```

Comment:
Not a big fan of passing pointers here. It would be more clear to write statements like `projectConfig.Context["ShouldPushRepoUpstream"] = promptPushRepoUpstream(projectConfig)`